src/renderer/renderer.ts

```diff
@@ -500,59 +525,60 @@ export class SDFGRenderer {
             }).appendTo(this.toolbar);
 
             // Overlays menu.
-            const overlayDropdown = $('<div>', {
-                class: 'dropdown',
-            });
-            $('<button>', {
-                class: 'btn btn-light btn-sdfv-light btn-sdfv',
-                html: '<i class="material-icons">saved_search</i>',
-                title: 'Overlays',
-                'data-bs-toggle': 'dropdown',
-                'data-bs-auto-close': 'outside',
-            }).appendTo(overlayDropdown);
-            const overlayMenu = $('<ul>', {
-                class: 'dropdown-menu',
-                css: {
-                    'min-width': '200px',
-                },
-            }).appendTo(overlayDropdown);
-            $('<div>', {
-                class: 'btn-group',
-            }).appendTo(this.toolbar).append(overlayDropdown);
-
-            const addOverlayToMenu = (
-                txt: string, ol: typeof GenericSdfgOverlay
-            ) => {
-                const olItem = $('<li>', {
+            if (!this.in_vscode) {
+                const overlayDropdown = $('<div>', {
+                    class: 'dropdown',
+                });
+                $('<button>', {
+                    class: 'btn btn-light btn-sdfv-light btn-sdfv',
+                    html: '<i class="material-icons">saved_search</i>',
+                    title: 'Overlays',
+                    'data-bs-toggle': 'dropdown',
+                    'data-bs-auto-close': 'outside',
+                }).appendTo(overlayDropdown);
+                const overlayMenu = $('<ul>', {
+                    class: 'dropdown-menu',
                     css: {
-                        'padding-left': '.7rem',
+                        'min-width': '200px',
```

Comment:
what does this do? effectively

Reply:
The padding left was just unnecessary and has thus been removed (looked bad). Without `min-width` the whole thing for some reason was pulled together such that something like "Storage Location" was shown with a line break between the words. `min-width` just ensures that no such silly width calculations are performed and we always have at least a reasonable 200px to work with (more if overlay names are longer)